packages/material-ui-lab/src/Pagination/Pagination.js

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { withStyles } from '@material-ui/core/styles';
+import usePagination from './usePagination';
+import PaginationItem from '../PaginationItem';
+
+export const styles = {
+  /* Styles applied to the root element. */
+  root: {
+    display: 'flex',
+    flexWrap: 'wrap',
+    alignItems: 'center',
+    listStyleType: 'none',
+    padding: 0, // Reset
+    margin: 0, // Reset
+  },
+};
+
+const Pagination = React.forwardRef(function Pagination(props, ref) {
+  const {
+    children,
+    classes,
+    className,
+    color = 'standard',
+    component: Component = 'ul',
+    getItemAriaLabel: getAriaLabel,
+    items,
+    renderItem = item => <PaginationItem {...item} />,
+    shape = 'round',
+    size,
+    variant = 'text',
+    ...other
+  } = usePagination(props);
```

Comment:
I would propose that we split the handling of the default props and the `usePagination`. Otherwise, it gets confusing to understand what props come from where. I have the Autocomplete pattern in mind. What do you think about it?

Reply:
One could argue similarly about passing all the props to useAutocomplete (and usePagination), and not knowing what props they actually consume, but I'll match useAutocomplete for consistency.

Reply:
Actually, I haven't tried this approach with the autocomplete. Maybe it's better. What about we don't change it here so we can try both out?

Reply:
Sure - it's a simple enough change if we need to make it.